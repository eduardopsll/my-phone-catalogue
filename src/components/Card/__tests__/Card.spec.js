import React from "react";
import ReactDOM from "react-dom";

import { Card } from "../index";

jest.mock("react-redux");

describe("Card", () => {
  it("should render without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Card />, div);
  });
});
