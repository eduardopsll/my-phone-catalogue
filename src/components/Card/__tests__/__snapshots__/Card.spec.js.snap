// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Param id undefined should render an empty component if there is not item selected 1`] = `<div />`;

exports[`Card Param id undefined should render the data of the item selected 1`] = `
<div>
  <div
    class="card"
  >
    <div
      class="card__close"
    >
      <img
        alt="close phone selected"
        class="card__close-icon"
        data-testid="closeCard"
        src="close.svg"
      />
    </div>
    <div
      class="card__info"
    >
      <img
        alt="Galaxy S7 Samsung"
        class="card__image"
        src="/images/Galaxy_S7.png"
      />
      <div
        class="card__text"
      >
        <div
          class="characteristics"
        >
          <h1
            class="card__title"
          >
            Galaxy S7
          </h1>
          <p
            class="card__subtitle"
          >
            Samsung
          </p>
          <p
            class="card__characteristic"
          >
            Screen: 
            5,1 inch Quad-HD
          </p>
          <p
            class="card__characteristic"
          >
            Processor: 
            Snapdragon 820
          </p>
          <p
            class="card__characteristic"
          >
            Ram: 
            4
          </p>
          <p
            class="card__characteristic"
          >
            Color: 
            <span
              class="card__color-sample"
              style="background-color: grey;"
            />
             
            grey
          </p>
        </div>
        <p
          class="description"
        >
          Introducing the smartphone your life can not do without, The Samsung Galaxy S7. Packed with features to keep you both productive and entertained, all in a sleek, slim design that fits comfortably in your hand or pocket.
        </p>
      </div>
    </div>
  </div>
</div>
`;
